guides/runbooks/orca-quality-of-service.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: single
+title:  "Orca Quality of Service"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+**EXPERIMENTAL**: This feature is still in an early adoption / experimental phase. 
+While you can use it today (Orca v6.71.0), Netflix is still currently running this in learning mode / judiciously enabling in response to on-call events.
+
+Spinnaker ships with an optional Quality of Service (QoS) module that can be used to manage the amount of active executions running at any given time.
+By default, this QoS module is disabled, but can be enabled and tuned with a handful of knobs and different strategies.
+Before we dive into the configuration settings, we'll go over how QoS works.
+
+# QoS Internals
+
+The QoS system is self-contained within Orca's [orca-qos][module] module and operates only on newly created executions.
```

Comment:
are these pipeline executions or does it also include orchestrations, make it more explicit